selectlinestatusbar.js

```diff
@@ -7,14 +7,24 @@ const
 
 class SelectLineStatusBar {
   constructor() {
-    this._statusBar = vscode.window.createStatusBarItem();
-    vscode.window.onDidChangeActiveTextEditor(e => this.displaySelectedLineCount(e.selections));
-    vscode.window.onDidChangeTextEditorSelection(e => this.displaySelectedLineCount(e.selections));
-    vscode.window.onDidChangeTextEditorViewColumn(e => this.displaySelectedLineCount(e.selections));
+    let alignment = vscode.StatusBarAlignment.Left;
+    let alignConfig = vscode.workspace.getConfiguration('selectline').alignment;

```

Comment:
It seems that we should restart VSCode for these settings to be reflected.
I think it is better to configure these while checking the change result every save the `settings.json`.